src/js/BookReader/Mode2Up.js

```diff
@@ -408,7 +428,7 @@ export class Mode2Up {
     }
 
     // $$$ check this logic with large spreads
-    ideal.reduce = ((first.height + second.height) / 2) / ideal.height;
+    ideal.reduce = Math.round(((first.height + second.height) / 2) / ideal.height);
```

Comment:
i pinned to an integer. the floats were too long, and it made for better comparison